src/actions/guides/testing/testing_actions.cr

```diff
@@ -0,0 +1,127 @@
+class Guides::Testing::TestingActions < GuideAction
+  guide_route "/testing/testing-actions"
+
+  def self.title
+    "Testing Actions"
+  end
+
+  def markdown : String
+    <<-MD
+    ## Setup
+
+    Located in your app's `spec/support/` directory is an `AppClient` that you can use to make requests
+    to your app's Action classes. Use this object to set any custom headers, or params you need.
```

Comment:
I have a lot of browser actions that do some pretty complex things to look for specific data based on params being passed over and such. I don't think Flow really tests that sort of stuff. I look at Flow like testing views only. 

For example, we have a lot of actions that look like this:

```crystal
  get "/favorites/:member_id" do
    favorites = InteractionsReactionQuery.new
      .faborable
      .member_id(member_id)
    total_count = favorites.clone.select_count
    favorites = favorites.paginate(page: page, per_page: per_page)
      .map(&.video_id.to_i32)

    releases = ReleaseQuery.new.release_id.in(favorites)
    render_component(VideoReleaseList,
      releases: VideoDecorator.new(releases, {access: :guest}),
      network_display: false,
      member_access_level: current_member_access_level,
      html_options: {"class" => "favorite", "data-count" => (total_count / per_page).ceil.to_i.to_s})
  end
```

So this doesn't return json, and it doesn't render a page.

I can make this all about testing API actions for now since we don't really have a clear way on testing non json actions, but I think in the future we should have a clear way (as we previously talked about).